symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -36,6 +36,7 @@ public function registerBundles()
 
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
+        // Feel free to remove the 1st parameter if you are using Symfony 4.0+
```

Comment:
I would rather put the name instead of saying `1st`. It will be more robust (especially if someone removes it without removing the comment)